BedrockServer.cpp

```diff
@@ -63,7 +63,7 @@ void BedrockServer::acceptCommand(SQLiteCommand&& command, bool isNew) {
 }
 
 void BedrockServer::cancelCommand(const string& commandID) {
```

Comment:
why leave this in at all?

Reply:
Cause I have to touch a bunch of stuff to remove it (SQLiteNode expects it to exist in SQLiteServer).